frontend/src/pages/ShowProfile.tsx

```diff
@@ -0,0 +1,97 @@
+import { Descriptions, DescriptionsProps } from 'antd';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
```

Comment:
See my later review comment for context
```suggestion
import { fetchProfile } from '../services';

```